src/Buttons/__tests__/Button.test.tsx

```diff
@@ -63,4 +66,46 @@ describe('Button', function () {
       assert.strictEqual(clickSpy.callCount, 0);
     });
   });
+  context('when forwardRef prop is present', function () {
+    beforeEach(function () {
+      const ButtonRefExample = () => {
+        const ref = useRef<HTMLInputElement>(null);
+        const onButtonClick = () => {
+          setTimeout(() => {
+            if (ref.current) {
+              ref.current.focus();
+            }
+          });
+        };
+        return (
+          <>
+            <Button
+              onClick={onButtonClick}
+              variant={VARIANT.PRIMARY}
+            >
+              Focus the Other Button
+            </Button>
+            <Button
+              onClick={(): void => {}}
+              variant={VARIANT.DANGER}
+              tabIndex={0}
+              forwardRef={ref}
+            >
+              Danger Button
```

Comment:
Trivial change : instead of `Danger Button` I prefer something like `borderless button` because the child is not always a danger (trash) button.

Reply:
This was also updated through a5e21f1.